.github/workflows/src/update-labels.js

```diff
@@ -20,6 +20,12 @@ export default async function updateLabels({ github, context, core }) {
     run_id = run_id || inputs.run_id;
   }
 
+  if (Number.isNaN(issue_number) || Number.isNaN(run_id)) {
+    core.info(`skipping run because issue_number or run_id is not a valid number. issue_numer: ${issue_number}, run_id: ${run_id}`);
+    return;
+  }
+  const pullRequestUrl = `https://github.com/${owner}/${repo}/pull/${issue_number}`;
+  core.info(`pull request url: ${pullRequestUrl}`);
```

Comment:
```suggestion
```

I think all this code can be deleted.  The decision on how to handle undefinded issue_number should be deferred as late as possible, until we know if it's necessary or not.

Reply:
Why not return as early as possible?

Reply:
Reported error in the later steps.